Java-Majasdarbi/src/main/java/advanced/test/Exercise1/NotEnoughFundsException.java

```diff
@@ -0,0 +1,13 @@
+package advanced.test.Exercise1;
+
+public class NotEnoughFundsException extends RuntimeException {
+    private String message;
+
+    public NotEnoughFundsException(String s) {
+        message = s;
```

Comment:
Te varēja saukt super konstruktoru un tad attiecīgi tas mainīgais message kļūtu lieks, jo runtime exception arī sevī iekšā to message saglabā, lai pēc tam varētu izgūt.